profile/profile.md

```diff
@@ -0,0 +1,71 @@
+# Roshan Ramesh . Jarvis Consulting
+
+Experienced recent Electrical Engineering graduate with a knack for creative Problem solving. Skills include delegation, organizational, time-management with prioritizing workflow for optimal throughput. Disciplined, encouraging, strong communicator and a warm extrovert with the ability to bond among introverts with ease. As an Electrical Engineering Graduate aiming to shift into Software(Data), the Jarvis training program promised exactly what was required to hone the data engineering skills through its rigourous industry standard mentoring. Keen to apply learned skills from jarvis in software and IT to improve client, users, employees and administration QOE.
+
+## Skills
+
+**Proficient:** Java/Lambda & Stream API, Linux/Bash, SQL, Matlab, Wireshark & Simulink, Agile/Scrum, Git
+
+**Competent:** RDBMS, Springboot, Mockito/Junit, REST APIs, Python, C++/C, Computer Networking
+
+**Familiar:** Google Cloud Platform, Visual Studios, VHDL, ARM Assembly, Machine Learning, Spark & Hadoop, HTML
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_demo](https://github.com/jarviscanada/jarvis_data_eng_demo)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_demo/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_RoshanRamesh/tree/master/linux_sql)]: Developed a real time cluster monitoring bash agent that collects and records hardware specifications and node usage information(CPU/memory). PostgreSQL database was used to store information of all nodes, provisioned by docker containers. Cronjob was used for automatic resource usage collection and SQL queries were written to account for business questions.
```

Comment:
Developed a real-time cluster monitoring bash agent with **Bash scripts** and SQL.

mention how you used docker

be clear on what kind of biz questions you have answered